named-entity-recognition/README.md

```diff
@@ -13,16 +13,34 @@ There are some sources for labeled NER data that can be used to train systems.
 
 You can also create data yourself if it doesn't exist.
 
-TODO: A good procedure to do this should be filled in.
+* Start by collecting unlabelled texts from popular domains like News articles or Wikipedia. If unlabelled texts do not exist, you can translate some sentences. 
+* Start with few sets of entities especially the popular ones like Personal names (PER), organization (ORG), location (LOC) and/or geopolitical entities (GPE). With few sets of entities, one can easily ensure high quality annotation since fine-grained entities and numerous entities can affect the quality of the overall process especially for non-expert annotators. 
```

Comment:
Personal names
-> personal names

few sets of entities
-> a smaller set of entities

one can easily ensure high quality annotation
-> it is easier to maintain the quality of annotation